tests/core/test_validation.py

```diff
@@ -0,0 +1,91 @@
+from argparse import Namespace
+from pathlib import Path
+
+import pytest
+
+from ethpm_cli.exceptions import InstallError, UriNotSupportedError, ValidationError
+from ethpm_cli.validation import validate_cli_args
+
+
+@pytest.fixture
+def args():
+    namespace = Namespace()
+    setattr(namespace, "uri", "ipfs://QmbeVyFLSuEUxiXKwSsEjef6icpdTdA4kGG9BcrJXKNKUW")
+    setattr(namespace, "ethpm_dir", None)
```

Comment:
Any reason you are using `setattr` here instead of just assigning to the attribute?